# Allele-Specific DNA Methylation Across Social Insects

Imprinted genes are genes which show parent-of-origin allele-specific expression. In mammals these genes are marked with epigenetic modifcations which can functionally repress expression in the imprinted copy. Often the epigenetic modification is promotor DNA methylation. It is predicted by the kinship theory that imprinted genes should exist in social insects. Recent literature has identified a small number of parent-of-origin DNA methylation marks in the honeybee (REF) and general allele-specific DNA methylation differences between reproductive and sterile workers of the bumblebee (REF). Also, genes which show parent-of-origin expression have been shown to exist in both honeybees and bumblebees (REFS).

The aim of this study is to determine how widespread allele-specific DNA methylation is across social insects to determine it's feasability as an imprinting mark. We hypothesise allele-specific DNA methylation will be more prevelant in highly eusocial species compared to less social / solitary species.

We will use previously generated data from 7 species (3 bees, 3 ants and 1 wasp) with varying levels of sociality to search for allele-specific DNA methylation (ASM). We have chosen these sampels as they all originate from female whole body tissue (REF). We will identify where ASM occurs throughout the genome, e.g. mostly genes/exons? We will compare the ASM patterns between highly social and less social / solitary species to see if ASM is associated with sociality (taking into account phylogenetic distance). We will also carry out reciprocal blasts to determine if orthologs of the same genes share ASM across species and we will carry out GO enrichment analysis to get some idea of the function of genes with ASM across social species.

NOTE: Allele-specific DNA methylation has been widely shown to be dependent on the underlying genotype (REFS e.g. Mikhail Matz new paper coming soon) and so we will pool genetically distinct replicates to decrease the effect of genotype (but we can't rule it out completely).

---

Project structure here

---

**Hollie NOTES:**
We have ran up to DAMEfinder to identify ASM in all species. Now need to annotate the ASM positions with genomic annotations (e.g. gene/exon). 

**THOUGHTS:** is the data really good enough to answer our question. I'm thinking maybe not...
- whole body not great
- some females are queens others workers
- not all have lambda/ecoli spike (BIG ISSUE)
- not that many species over all (not many solitary / less social)
- crappy genomes so probably poor gene annotations
- don't have a good way to account for genotype (THIS IS THE BIGGEST ISSUE I THINK)

Probably we would be better to win a small grant to do this properly, although WGBS of say 10 species WITH genetically different replicates could be an issue, although you would have to pool anyway because using one tissue (head/brain) won't yield enough. Although you would still want biological replicates of pool (at least 3).

Speak with Alun/Laura/Eamonn, even though we've invested time in this, overall it's not that much time and I don't want to do crappy science. 